docs/book/auth-adapter.md

```diff
@@ -0,0 +1,48 @@
+# Authentication adapters
+
+The authentication adapters for `zend-expressive-authentication` implement the
+interface `AuthenticationInterface` reported below:
```

Comment:
I think the full name for the interface would be good: `Zend\Expressive\Authentication\AuthenticationInterface`

Reply:
@froschdesign thanks for the feedback. I also used the full namespace interface in other files.